domains/yashasviallen.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "JavaProgswing",
+        "email": "yashasviallen@gmail.com"
+    },
+    "record": {
+        "CNAME": "yashasviallen.dynv6.net"
```

Comment:
site cant be reached

Reply:
It's reachable from outside for me, beware site is ipv6.

Reply:
@CuteDog5695 site is on ipv6, only accessible from ipv6 clients.

Reply:
so basically not everyone can see the website?

Reply:
@CuteDog5695 Yes I cannot port an cloudflare proxy on the same cuz there are things that would break with a proxy.
So it'll remain open to ipv6 clients only.

Reply:
@CuteDog5695 Can you resolve this please? I cannot get a ipv4 site atm. This site fully works on ipv6.